includes/discovery/processors/cambium.inc.php

```diff
@@ -21,7 +21,7 @@
             '0',
             'cambium-cpu',
             $descr,
-            '100',
+            '10',
```

Comment:
Thanks for confirming. @neg2led can you remove the polling file as I think it's not needed.